pwa/src/components/applications/applicationForm.tsx

```diff
@@ -0,0 +1,233 @@
+import * as React from "react";
+import {
+  GenericInputComponent,
+  Spinner,
+  Card,
+  Alert,
+  Accordion
+} from "@conductionnl/nl-design-system/lib";
+import { isLoggedIn } from "../../services/auth";
+import { Link } from "gatsby";
+import { navigate } from "gatsby-link";
+import {
+  checkValues,
+  removeEmptyObjectValues, retrieveFormArrayAsOArray,
+} from "../utility/inputHandler";
+import {ArrayInputComponent} from "../common/arrayInput";
+import FlashMessage from 'react-flash-message';
+
+export default function ApplicationForm({ id }) {
+  const [context, setContext] = React.useState(null);
+  const [alert, setAlert] = React.useState(null);
+  const [application, setApplication] = React.useState(null);
+  const [showSpinner, setShowSpinner] = React.useState(false);
```

Comment:
Ik mis hier bij alle `useState` definitions de type, dat zou in het geval van de `showSpinner` een `boolean` zijn en als volgt aangeduid worden: `const [showSpinner, setShowSpinner] = React.useState<boolean>(false);`